packages/element-lib/src/__tests__/__fixtures__/didDocumentWithCustomContext.json

```diff
@@ -0,0 +1,40 @@
+{
+  "@context": "https://docs.element-did.com/contexts/sidetree/sidetree-v0.1.jsonld",
```

Comment:
This needs to be an array, with the method specific context in the second position for interop.

Reply:
It is an array with the method specific context when you resolve it: https://docs.element-did.com/contexts/sidetree/sidetree-v0.1.jsonld

Reply:
See the did core spec on extending context with method specific context